cards/src/main/resources/cards/custom/group10/spellsource_core_legendary/minion_matriarch_aiiran.json

```diff
@@ -28,16 +28,26 @@
   },
   "dynamicDescription": [
     {
-      "class": "ValueDescription",
-      "value": {
-        "class": "CardCountValueProvider",
-        "cardFilter": {
-          "class": "CardFilter",
-          "cardType": "MINION",
-          "race": "DRAGON"
-        },
-        "multiplier": 2,
-        "offset": -2
+      "class": "ConditionalDescription",
+      "condition": {
```

Comment:
this is probably the wrong condition for this. maybe you want to check if it's on the battlefield?

Reply:
I believe that's what this Condition is checking for. I've added a test that checks that the description changes appropriately.